packages/legend-graph/src/graphManager/AbstractPureGraphManager.ts

```diff
@@ -195,6 +196,10 @@ export abstract class AbstractPureGraphManager {
     valueSpecificationJson: Record<PropertyKey, unknown>,
     graph: PureModel,
   ): ValueSpecification;
+  abstract buildLambdaParameters(
```

Comment:
Dont think this function is needed. You should be able to leverage the `buildValueSpecification` function. Reading and asserting the type of value specification needed for this.